### PR TITLE
Jb/tags result page

### PR DESCRIPTION
**Pull request by jbouzi12**
This is a WIP PR just to combine the routes portion of the tags results page with the template. 
I set up the page with what the shape of the data and the content, but still need to tweak things after hooking this up to the routes.

### CODE REVIEW

**Comment by jbouzi12** [comment]
Not sure if this PR is the best way to combine the two so Jordan and I can make final tweaks once both of these are together. What do you guys think? I'm ok with just merging this with @jestapinski 's  branch so the page can actually render, then we both can make the necessary updates to finalize the experience.

**Comment by jbouzi12** [comment]
Still have to fix the merge conflict and tweak some padding but the pages render with articles now!

**Comment by jestapinski** [comment]
> Still have to fix the merge conflict and tweak some padding but the pages render with articles now!

Try re-pulling my branch and rebasing again! Made some small tweaks

**Comment by jbouzi12** [comment]
@jestapinski @greggb This PR is good to go now. At the moment the `bio`, `location`, and `title` have yet to be included in our articles yet. But I conditionally render them for authors in anticipation for these additions.

File: src/templates/tag.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+            <HeroBanner
+                breadcrumb={[
+                    { label: 'Home', to: '/' },
+                    { label: 'Learn', to: '/learn' },
+                    { label: capitalizedBreadcrumb, to: slug },
+                ]}
+            >
+                {!isAuthor && (
+                    <TagTitle>
+                        <P>TAGGED IN</P>
```
Comment (jestapinski) [inline src/templates/tag.js]: It looks like the designs want this text a little bit smaller, would you mind doing another check here?

File: src/templates/tag.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+            slug,
+            title,
+            type,
+        },
+    } = props;
+    const isAuthor = type === 'author';
+    const articles = constructArticles(pages);
+    const capitalizedBreadcrumb = name.charAt(0).toUpperCase() + name.slice(1);
+    return (
+        <Layout>
```
Comment (jestapinski) [inline src/templates/tag.js]: I'm still seeing some weird issues with the header not sticking to the top (on my monitor specifically it seems), do we know what might be causing this?

Screen Shot 2020-02-28 at 10 40 36 AM

Comment (jbouzi12) [reply src/templates/tag.js]: Hmm yea I noticed that on your screen before but I'm not seeing this on mine. Is your screen zoomed in?

**Comment by jbouzi12** [comment]
@jestapinski I pushed up a fix for the spacing issues. Let me know what you think.

**Pull request merged**